File: skills/quality/dry-test/fixtures/csharp/Ops.cs
// Word operators, including the negated pattern form.
class Ops {
    bool WordA(object a, object b) { return a is string; }

    bool WordB(object a, object b) { return a is not string; }

    bool WordC(object a, object b) { return a as string != null; }
}

File: skills/quality/dry-test/fixtures/kotlin/ops.kt
// Word operators, and the negated forms the grammar spells with a leading `!`.
fun wordA(a: Any, b: List<Any>): Boolean = a in b

fun wordB(a: Any, b: List<Any>): Boolean = a !in b

fun wordC(a: Any, b: List<Any>): Boolean = a is String

fun wordD(a: Any, b: List<Any>): Boolean = a !is String
